metrics/nodehealthcheck.go

```diff
@@ -42,10 +44,19 @@ var (
 	)
 )
 
+var nodehealtCheckRemediationDuration = prometheus.NewHistogramVec(
+	prometheus.HistogramOpts{
+		Name:    "nodehealthcheck_remediation_duration_seconds",
+		Help:    "Remediation duration distribution",
+		Buckets: []float64{30, 120, 300, 1500, 1800, 3600},
```

Comment:
How fast can a remediation be? Can it be shorter than 30 seconds?
Is it makes sense to a have wider delta between 1500 and 1800 and have more buckets in the short time than long time (above 1000 seconds)?
Maybe  use `30, 60, 120, 180, 360, 900, 1800, 3600`.
I personally likes a "jump of two (x2)", rather than vary offset between numbers.

Reply:
> "jump of two (x2)"

that sounds reasonable in most cases, but our customers are very interested in low times, so here it makes sense to have some more values in the e.g. <5 min frame.

What about 30, 60, 120, 180, 240, 300, 600, 1200, 2400, 3600 🤔

Reply:
It does make sense to me